astropy/table/table.py

```diff
@@ -878,6 +878,10 @@ def show_in_notebook(self, tableid=None, css=None, display_length=50,
             to ``astropy.table.jsviewer.DEFAULT_CSS_NB``.
         display_length : int, optional
             Number or rows to show. Default to 50.
+        add_row_index : bool
```

Comment:
Maybe using the word `show` instead of `add` here might make it more obvious that nothing is happening to the table itself, and instead this is just a feature of how the table is displayed.